src/components/MapboxMap.js

```diff
@@ -314,7 +314,15 @@ class MapboxMap extends Component {
             <MapProperties center={lngLat} map={this.map} />
           )}
           {/* Markers */}
```

Comment:
I think it might be worth saying in the comment that this includes data group markers

Reply:
And saying in the comment for the above MapDataGroups component that it excludes markers, due to the clustering issue

Reply:
Done